### PR TITLE
i18n(es): add microsoft azure

### PR DESCRIPTION
**Pull request by fredoist**
#### What kind of changes does this PR include?
- New or updated content
- Translated content

#### Description
- What does this PR change?  Adds Microsoft Azure deploy guide in Spanish

### CODE REVIEW

File: src/pages/es/guides/deploy/microsoft-azure.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Despliega tu proyecto de Astro en Microsoft Azure
+description: Cómo desplegar tu proyecto de Astro usando Microsoft Azure
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+[Azure](https://azure.microsoft.com/) es una plataforma de informática en la nube de Microsoft. Puedes desplegar tu proyecto de Astro con el servicio de [Aplicaciones Web Estáticas](https://aka.ms/staticwebapps) de Microsoft.
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/microsoft-azure.md:8]:
```suggestion
[Azure](https://azure.microsoft.com/) es una plataforma en la nube de Microsoft. Puedes desplegar tu proyecto de Astro con el servicio de [Aplicaciones Web Estáticas](https://aka.ms/staticwebapps) de Microsoft.
```

File: src/pages/es/guides/deploy/microsoft-azure.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- Subir el código de tu proyecto a [GitHub](https://github.com/).
+- La [Extensión de SWA](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestaticwebapps) en [Visual Studio Code](https://code.visualstudio.com/).
+
+## Cómo desplegar
+
+1. Abre tu proyecto en VS Code.
+
+2. Abre la extensión de Aplicaciones Web Estáticas, inicia sesión en Azure, y haz clic en el botón de **+** para crear una nueva Aplicación Web Estática. Se te pedirá que designes la clave de subscripción a usar.
+
+3. Sigue el asistente iniciado por la extensión para darle a tu aplicación un nombre, seleccionar un framework preestablecido, y designar la raíz de tu aplicación (usualmente `/`) y la ubicación de los archivos generados `/dist`. El asistente creará y ejecutará una [GitHub Action](https://github.com/features/actions) en tu repositorio dentro del directorio `.github`.
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/microsoft-azure.md:24]:
```suggestion
3. Sigue el asistente iniciado por la extensión para darle a tu aplicación un nombre, seleccionar un framework preestablecido y designar la raíz de tu aplicación (usualmente `/`) y la ubicación de los archivos generados `/dist`. El asistente creará y ejecutará una [GitHub Action](https://github.com/features/actions) en tu repositorio dentro del directorio `.github`.
```

File: src/pages/es/guides/deploy/microsoft-azure.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+La GitHub Action desplegará tu aplicación (puedes ver el progreso en la pestaña de Actions en GitHub). Cuando se complete con éxito, puedes ver tu aplicación en la dirección mostrada en la ventada de progreso de la Extensión de SWA haciendo clic en el botón de **Browse Website** (esto aparecerá después de que la GitHub Action se haya ejecutado).
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/microsoft-azure.md:26]:
```suggestion
La GitHub Action desplegará tu aplicación (puedes ver el progreso en la pestaña de Actions en GitHub). Cuando se complete con éxito, puedes ver tu aplicación en la dirección mostrada en la ventana de progreso de la Extensión de SWA haciendo clic en el botón de **Browse Website** (esto aparecerá después de que la GitHub Action se haya ejecutado).

```

**Review by agustinmulet — COMMENTED** [review]
Muy buen trabajo! Ya falta menos 😤

**Comment by Yan-Thomas** [comment]
Good job @fredoist, @agustinmulet! Thanks for this :raised_hands:

**Pull request merged**